macros/edr/tests/test_exposure_schema_validity.sql

```diff
@@ -0,0 +1,64 @@
+{% test exposure_schema_validity(model, exposures, node, columns) %}
+    {%- if not execute -%}
+        {%- do return(none) -%}
+    {%- endif -%}
+
+    {# Parameters used only for dependency injection in integration tests #}
+    {%- set node = node | default(context['model']['attached_node']) -%}
```

Comment:
Can you replace the `default()` to `or` operators?
Also, when accessing a dict value, you can use `.get(key, default)`.

Reply:
replaced to `or`